src/styles/.blocks/variables.css

```diff
@@ -0,0 +1,15 @@
+:root {
+
+  --light-bg-color: #fff;
+  --midium-bg-color: #e5e5e5;
```

Comment:
There is a typo in the variable name, change --midium-bg-color to --medium-bg-color.